src/uv.cc

```diff
@@ -60,15 +60,12 @@ using v8::Value;
 
 void ErrName(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
-  if (env->options()->pending_deprecation && env->EmitErrNameWarning()) {
-    if (ProcessEmitDeprecationWarning(
-        env,
-        "Directly calling process.binding('uv').errname(<val>) is being"
-        " deprecated. "
-        "Please make sure to use util.getSystemErrorName() instead.",
-        "DEP0119").IsNothing())
-    return;
-  }
+  if (ProcessEmitDeprecationWarning(
```

Comment:
It would be more consistent to wrap `internalBinding('uv').errname` with the deprecation warning here instead:

https://github.com/nodejs/node/blob/ab3f032310da7d5c5539a6c199eaa5e188433c5e/lib/internal/bootstrap/pre_execution.js#L223